src/estimation/src/BerdyHelper.cpp

```diff
@@ -125,6 +125,17 @@ bool BerdyHelper::init(const Model& model,
     m_sensors = sensors;
     m_options = options;
 
+    if (!options.baseLink.empty()) {
+        //find the LinkIndex corresponding to the baseLink option
+        LinkIndex baseLinkIndex =  m_model.getLinkIndex(options.baseLink);
+        
+        if (baseLinkIndex == LINK_INVALID_INDEX
+            || !m_model.setDefaultBaseLink(baseLinkIndex)) {
```

Comment:
I would like to keep the "default base" to always mean "the default base as loaded from the model". For changing the base of `m_dynamicsTraversal`, we can simply pass the base link as a parameter to `computeFullTreeTraversal`.